cnd/src/http_api/markets/get_btc_dai.rs

```diff
@@ -24,7 +24,7 @@ async fn handler(facade: Facade) -> Result<impl Reply> {
     let mut orders = siren::Entity::default();
     let local_peer_id = facade.swarm.local_peer_id();
 
-    for (maker, order) in facade.swarm.btc_dai_market().await {
+    for (maker, order) in facade.swarm.btc_dai_market_safe_expiries().await {
```

Comment:
I think on this level I would probably just imply that all the orders are safe and just retain the name "btc_dai_market".